### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Rishi-Mishra0704**
### Views

- Add basic structure for user/index.html.erb
- Add basic structure for user/show.html.erb
- Add basic structure for post/index.html.erb
- Add basic structure for post/show.html.erb
- Add styles for every page as per the wireframe

### CODE REVIEW

File: db/migrate/20230627131850_add_default_to_posts_counter_in_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddDefaultToPostsCounterInUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_column_default :users, :posts_counter, 0
+  end
+end
```
Comment (zieeco) [inline db/migrate/20230627131850_add_default_to_posts_counter_in_users.rb:5]: - Hey, this migration file is causing your app to throw an error. please, consider deleting the file as it does not exist.

image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Rishi-Mishra0704,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice views. ✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230627131850_add_default_to_posts_counter_in_users.rb (hunk 1 above)
Comment (Rishi-Mishra0704) [reply db/migrate/20230627131850_add_default_to_posts_counter_in_users.rb:5]: @zieeco , Hey, I think you have mixed the names, please specify more clearly

Comment (zieeco) [reply db/migrate/20230627131850_add_default_to_posts_counter_in_users.rb:5]: @Rishi-Mishra0704, that particular one, delete it. [this does not exit](https://github.com/Rishi-Mishra0704/blog-app/blob/cc6dd67343767bf70cfe0da906b50ab9104baf41/db/migrate/20230620124654_rename_author_id_to_user_id.rb#L1-L6)

**Review by Davidosky007 — APPROVED** [review]
Hi @Rishi-Mishra0704 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR 👍 
* All views displayed properly 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**